test/index.js

```diff
@@ -1,24 +1,84 @@
+'use strict';
+
 const assert = require('assert');
 const nock = require('nock');
-const {verify} = require('../index');
 
-// mock the request
-nock('https://hcaptcha.com')
-  .post('/siteverify', 'secret=secret&response=token')
-  .reply(200, '{"success":true}')
+nock.disableNetConnect();
```

Comment:
Why the call to nock before the require? Do we need to call `nock.disableNetConnect` before requiring the index.js file in order for the mocking to work?

Reply:
order should not matter, i'll swap it.